packages/synapse-react-client/src/components/Forum/DiscussionReply.tsx

```diff
@@ -73,7 +73,12 @@ export function DiscussionReply(props: DiscussionReplyProps) {
   }, [reply.id])
 
   return (
-    <div className="reply-container" id={`reply._${reply.id}`} ref={replyRef}>
+    <div
+      className="reply-container"
+      id={`reply._${reply.id}`}
+      ref={replyRef}
+      role={'article'}
```

Comment:
Labeling the thread & replies as articles improves a11y (making it easier to write queries for e2e tests)